lib/unleash/strategies.rb

```diff
@@ -23,7 +23,13 @@ def fetch(name)
     end
 
     def add(strategy)
-      @strategies[strategy.name] = strategy
+      if default_strategy_names.include?(strategy.name)
+        # rubocop:disable Style/StderrPuts
```

Comment:
I personally would be consistent with the rest of the SDK use the logger for this (but use the error log level). And if you strongly believe in throwing this (and other errors) in stderr, configure the default logger to do so.

I am at least a bit fan on being consistent across all of the sdk wrt to logging. 

WDYT?

Reply:
What I'm trying to avoid is someone setting a logger config that allows them to ignore or miss this error. I want this to make people sad enough to do something about it today because if you ignore this and upgrade, your toggles will just evaluate in a different way and you'll have no idea why.

Consistency that big of a concern given this won't live very long (I know I'm a bad person).

I'm more worried about not playing nicely with the logger and having this get missed in another way

Reply:
Totally understand the need to be pedantic. Maybe do both?

Hate myself for mentioning, but maybe add something in CAPS too?

If people have log shippers pre-configured, it could get missed if they do something weird with their collections. 

Just my 2c.

Reply:
Eh. I thought about it. I don't mind that much either way. If you disable errors then your life is going to be hard.

I've moved this to the logger

Reply:
I would have not added it as a warning either. 😛